sdk-tests/src/test/java/io/dapr/it/pubsub/http/PubSubIT.java

```diff
@@ -402,6 +404,58 @@ public void testPubSubTTLMetadata() throws Exception {
     daprRun.stop();
   }
 
+  @Test
+  public void testlongValues() throws Exception {
+    final DaprRun daprRun = closeLater(startDaprApp(
+        this.getClass().getSimpleName(),
+        SubscriberService.SUCCESS_MESSAGE,
+        SubscriberService.class,
+        true,
+        60000));
+    // At this point, it is guaranteed that the service above is running and all ports being listened to.
+    if (this.useGrpc) {
+      daprRun.switchToGRPC();
+    } else {
+      daprRun.switchToHTTP();
+    }
+
+    ConvertToLong value = new ConvertToLong();
+    value.setVal(590518626939830271L);
```

Comment:
Create a Random object to generate random values with `590518626939830271L` as seed. 
Create a Set<ConvertToLong> HashSet, 
Add `590518626939830271L` as the first element. 
Add more random long values upto NUM_MESSAGES size.